protocol_rfcs/collated-string-type.md

```diff
@@ -0,0 +1,175 @@
+# Collated String Type
+**Associated Github issue for discussions: https://github.com/delta-io/delta/issues/2894**
+
+This protocol change adds support for collated strings. It consists of three changes to the protocol
+
+* Collations in the table schema
+* Collated AddFile statistics
+* Domain metadata with active collation version
+
+--------
+
+> ***Add a new section in front of the [Primitive Types](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#primitive-types) section.***
+
+### Collations table feature
+
+Collations are a set of rules for how strings are compared. The are supported by the `collations` table feature. Collations do not affect how strings are stored. Collations are applied when comparing strings for equality or to determine the sort order of two strings. Case insensitive comparison is one example of a collation where case is ignored when string are compared for equality and the lower cased variant of a string is used to determine its sort order.
+
+Collations can be specified for all string fields in a table schema. It is also possible to store statistics per collation version. This is required because the min and max values of a column can differ based on the used collation or collation version.
```

Comment:
```suggestion
Each string field can have a collation, which is specified in the table schema. It is also possible to store statistics per collation version. This is required because the min and max values of a column can differ based on the used collation or collation version.
```